### DISCUSSION TITLE
Do MAA do IS expeditioner's joklumarkar ?

### DISCUSSION THREAD

**Discussion by LipChuanSui** [MaaAssistantArknights/MaaAssistantArknights#8490] [Q&A] [answered]
Do MAA do IS expeditioner's joklumarkar ?

**Comment by swablueme** [comment, marked as answer]
If you look at https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/8523 (new #8524) it displays Constrat's current progress with regards to IS4.  (assuming you are on global)
